src/app/_models/team.ts

```diff
@@ -11,4 +11,5 @@ export interface Team {
     course_registrations: CourseRegistration[];
     score: number;
     member_names: string[];
+    number_of_member: number;
```

Comment:
Why do we have `number of member`? can we not get it from `member_names.length`?